[32mINFO    [0m juju.model:model.py:2972 Waiting for model:
  mongos-k8s/0 [idle] active:
[32mINFO    [0m pytest_operator.plugin:plugin.py:834 Model status:

Model  Controller          Cloud/Region        Version  SLA          Timestamp
test   microk8s-localhost  microk8s/localhost  3.5.3    unsupported  01:36:20Z

App            Version  Status  Scale  Charm        Channel  Rev  Address         Exposed  Message
application             active      1  application             0                  no       
config-server           active      1  mongodb-k8s  6/edge    55  10.152.183.246  no       
mongos-k8s              active      1  mongos-k8s              0  10.152.183.145  no       
shard0                  active      1  mongodb-k8s  6/edge    55  10.152.183.38   no       Primary

Unit              Workload  Agent  Address       Ports  Message
application/0*    active    idle                        
config-server/0*  active    idle   10.1.125.207         
mongos-k8s/0*     active    idle   10.1.125.202         
shard0/0*         active    idle   10.1.125.208         Primary

[32mINFO    [0m pytest_operator.plugin:plugin.py:840 Juju error logs:

unit-config-server-0: 01:30:49 ERROR unit.config-server/0.juju-log config-server:4: Failed to add shard shard0 to the config server, error=OperationFailure("Authentication failed., full error: {'ok': 0.0, 'errmsg': 'Authentication failed.', 'code': 18, 'codeName': 'AuthenticationFailed', '$clusterTime': {'clusterTime': Timestamp(1728351048, 2), 'signature': {'hash': b'\\xa7\\xa7P\\tY\\xec3\\xedx\\x89\\xbaz\\xb8;\\x93Iu%sX', 'keyId': 7423211094023340057}}, 'operationTime': Timestamp(1728351048, 2)}")
unit-config-server-0: 01:30:49 ERROR unit.config-server/0.juju-log config-server:4: shard0 shard does not have the same auth as the config server.

[32mINFO    [0m pytest_operator.plugin:plugin.py:904 Forgetting main...